docs/dev/api/insights.md

```diff
@@ -2109,13 +2523,272 @@ Return an array of test cases (grouped by name) with statistical details.
+```
+
+</TabItem>
+<TabItem value="eu">
+
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request GET "https://api.eu-central-1.saucelabs.com/insights/v2/tests?org_id=<org_id>" | json_pp
+```
+
+</TabItem>
+</Tabs>
+
+#### Responses
+
+<table id="table-api">
+<tbody>
+  <tr>
+    <td><code>200</code></td>
+    <td colSpan='2'>Success.</td>
+  </tr>
+</tbody>
+<tbody>
+  <tr>
+    <td><code>422</code></td>
+    <td colSpan='2'>Validation Error.</td>
+  </tr>
+</tbody>
+</table>
+
+```jsx title="Sample Response"
+{
+  "items": [
+    {
+      "id": "abc123",
+      "name": "test1",
+      "source": "rdc",
+      "status": "complete",
+      "creation_time": "2023-06-05T14:47:21.209Z",
+      "modification_time": "2023-06-05T14:47:21.209Z",
+      "error": "<error>",
+      "passed": true,
+      "browser_normalized": "<broser>",
+      "os_normalized": "<os>",
+      "device_name": "<device>",
+      "device_group": "private",
+      "build": "<build>",
+      "automation_backend": "<automation_backend>",
+      "duration": 0,
+      "org_id": "<ord_id>",
+      "user_id": "<user_id>",
+      "team_id": "<team_id>",
+      "group_id": "<group_id>",
+      "tags": [
+        "<tag>"
+      ],
+      "owner": "<owner>",
+      "ancestor": "<ancestor>",
+      "start_time": "2023-06-05T14:47:21.209Z",
+      "end_time": "2023-06-05T14:47:21.209Z",
+      "deletion_time": "2023-06-05T14:47:21.209Z",
+      "is_expired": false
+    }
+  ],
+  "total": 5,
+  "statuses": {
+    "additionalProp1": 3,
+    "additionalProp2": 5,
+    "additionalProp3": 2
+  },
+  "max_duration": 10
+}
+```
+
+</details>
+
+---
+
+### Get Test Cases
+
+<details><summary><span className="api get">GET</span> <code>/v2/insights/&#123;source&#125;/test-cases</code></summary>
+<p/>
+
+Return an array of test cases (grouped by name) with statistical details.
+
+#### Parameters
+
+<table id="table-api">
+  <tbody>
+    <tr>
+     <td><code>source</code></td>
+       <td><p><small>| PATH | REQUIRED | STRING |</small></p><p>Return results only for tests run in virtual device cloud or real device cloud. Supported values are:</p><p>
+     <ul>
+      <li><code>rdc</code> - Real Device Cloud</li>
+      <li><code>vdc</code> - Virtual Device Cloud</li>
+    </ul></p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>org_id</code></td>
+       <td><p><small>| QUERY| REQUIRED | STRING |</small></p><p>Return results only for the specified <code>org_id</code>.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>interval</code></td>
+     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>
+     Relative date filter. Available values are: <code>1m</code> (1 month), <code>15m</code> (15 months), <code>1h</code> (1 hour), <code>6h</code> (6 hours), <code>12h</code> (12 hours), <code>1d</code> (1 day), <code>7d</code> (7 days), <code>30d</code> (30 days). Default value is <code>1d</code>
```

Comment:
When there is more than one available value, we should use a bulleted list so the description is easier to read.
```suggestion
     Relative date filter. Available values are: <ul><li><code>1m</code> (1 month)</li><li><code>15m</code> (15 months)</li><li><code>1h</code> (1 hour)</li><li><code>6h</code> (6 hours)</li><li><code>12h</code> (12 hours)</li><li><code>1d</code> (1 day)</li><li><code>7d</code> (7 days)</li><li><code>30d</code> (30 days)</li></ul> Default value is <code>1d</code>.
```